src/codegen/llvm.zig

```diff
@@ -393,7 +392,6 @@ const DataLayoutBuilder = struct {
                 .pref = pref,
                 .idx = idx,
             };
-            if (self.target.cpu.arch == .aarch64_32) continue;
```

Comment:
Thinking through the code here, I was not able to convince myself that this statement actually needed to be here. I can add back in equivalent logic that checks the architecture + ABI if I'm wrong, though.